packages/awareness/src/global.css

```
@@ -18,38 +18,67 @@
+  --color-awareness-activity-join: #0891b2;
+  --color-awareness-activity-leave: #64748b;
   --shadow-awareness: 0 10px 24px
-    color-mix(in srgb, CanvasText 10%, transparent);
+    color-mix(in srgb, CanvasText 12%, transparent);
+  --shadow-awareness-close: 0 2px 8px
+    color-mix(in srgb, CanvasText 14%, transparent);
 }
 
 .awareness-avatar {
   --awareness-avatar-size: 32px;
+  box-sizing: border-box;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)